content/what-is/top-iac-tools.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Top Infrastructure as Code Tools
+meta_desc: |
+     Explore top Infrastructure as Code (IaC) tools for cloud resource management. Find the best fit for your DevOps needs.
+type: what-is
+page_title: "Top Infrastructure as Code Tools"
+---
+
+Infrastructure as Code tools (IaC tools) let you automate the setup of your cloud resources without manually clicking around in your cloud web console. Instead of manually configuring resources, you write a script that specifies what you need, and the cloud provider sets it up for you. It's a great way to make infrastructure setup consistent and repeatable.
+
+But which tool should you use choose? We have our preferences and biases 😉 but let's go through the most popular and some less popular IaC tools to give you a lay of the land.
+
+We'll assess each tool based on the following criteria:
+
+| Criteria    | Description                                                                 |
+|-------------|-----------------------------------------------------------------------------|
+| Flexibility | Ability to abstract and create reusable components                          |
+| Multi-cloud  | Capability to manage resources across various cloud providers               |
+| State       | How the tool handles state tracking and management                          |
+| Integration | Compatibility with existing development workflows and CI/CD pipelines       |
+| Ecosystem   | Community strength, documentation quality, and support resources            |
+
+We'll cover modern multi-cloud tools, vendor and cloud-specific solutions, and some less common options.
+
+## Multi-Cloud Infrastructure as Code Tools
+
+First up, IaC tools that manage infrastructure across multiple cloud providers. You should pick one of these if you don't have any unusual requirements or legacy investments.
+
+### Pulumi
+
+This is the Pulumi website, so it's not an unbiased source of information. Still, Pulumi stands out as a versatile Infrastructure as Code (IaC) tool, offering the flexibility to manage infrastructure using a variety of programming languages.
```

Comment:
I like lamp-shading the fact that we are Pulumi. 

I assume if you end up on this page via a search, biggest thought is how could I possibly trust this, and so be up front about that.

Reply:
I agree w/ @toriancrane that this feels unnecessary to say. While I agree with the motivation, I think if we have already softened the impact by describing our evaluation criteria in a way that feels largely unbiased, and includes factors where we are weaker than competitors (e.g. ecosystem).

It makes sense to me though, for the same purpose, to lead with TF not Pulumi, because at this point they are the market leader by a vast margin. 

Generally speaking, I do agree that it's important to call out our bias explicitly in places where it may not be as as clearly from Pulumi as our `pulumi.com` product website... e.g. in Reddit comments, GitHub, syndicated/partner articles, etc.

Reply:
Changed!